### PR TITLE
API-410: Add doc for php client about family variant and product model creation

### PR DESCRIPTION
**Pull request by willy-ahva**

### CODE REVIEW

File: content/php-client/resources/product-models.md [hunk 1]
```diff
@@ -27,3 +27,36 @@ $client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAut
...
+              'locale' => null,
+              'scope' => null,
+              'data' => 'saddle'
+          ]
+      ]
+  ]
+]);
+```
+
+You can get more information about the expected format of the product values [here](/documentation/resources.html#product).
```
Comment (NolwennP) [inline content/php-client/resources/product-models.md]: "product model values" instead of "product values"

Comment (NolwennP) [reply content/php-client/resources/product-models.md]: I would even be more precise by saying:
"Product model values use the same format as the product values. If you want to know more, take a look at [here](/documentation/resources.html#product-values)"
Beware, the link given was wrong too.

File: content/php-client/resources/family-variants.md [hunk 2]
```diff
@@ -30,3 +30,34 @@ $client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAut
...
+$client->getFamilyVariantApi()->create('boots', [
+    'code' => 'boots_size_color',
+    'labels' => [
+        'en_US' => 'Boots by color and size'
+    ],
+    'variant_attribute_sets' => [
+        [
+            'level' => 1,
+            'axes' => ['size'],
+            'attributes' => [
```
Comment (NolwennP) [inline content/php-client/resources/family-variants.md]: All attributes on one line ? (like below)

File: content/php-client/resources/family-variants.md [hunk 3]
```diff
@@ -30,3 +30,34 @@ $client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAut
...
+
+### Create a family variant
+
+If the family variant does not exist yet, this method creates it, otherwise it throws an exception.
+
+```php
+$client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$client->getFamilyVariantApi()->create('boots', [
+    'code' => 'boots_size_color',
```
Comment (skeleton) [inline content/php-client/resources/family-variants.md]: `code` should be added as an argument of the `create` method like the other methods

File: content/php-client/resources/product-models.md [hunk 4]
```diff
@@ -27,3 +27,36 @@ $client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAut
...
+
+### Create a product model
+
+If the product model does not exist yet, this method creates it, otherwise it throws an exception.
+
+```php
+$client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$client->getProductModelApi()->create([
+  'code' => 'saddle_rain_boots',
```
Comment (skeleton) [inline content/php-client/resources/product-models.md]: same remark as below

**Pull request merged**